### ISSUE TITLE
Somehow let screenreaders know the public / unpublish button is a toggle

### ISSUE THREAD

**Issue by cjshawMIT** [CLIxIndia-Dev/OpenAssessmentsClient#109]

**Comment by cjshawMIT** [comment]
I think we have addressed this in PR #183 -- the publish / unpublish button is now labelled with the correct action, and the `aria-label` for the entire row includes the current state of the assessment (published or unpublished). So screenreader users should have access to the same information that sighted users have.
